FirebaseDatabase/CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 # v7.5.0
+- [added] Implmemented `queryStartingAfterValue` for FirebaseDatabase queries.
```

Comment:
As mentioned in the other PR, it might be worth mentioning the pagination use case here.